web/src/__tests__/async/comments-api.servertest.ts

```diff
@@ -9,34 +9,39 @@ import {
   GetCommentV1Response,
   PostCommentsV1Response,
 } from "@/src/features/public-api/types/comments";
-import { PostTracesV1Response } from "@/src/features/public-api/types/traces";
 import { prisma } from "@langfuse/shared/src/db";
 import { z } from "zod";
+import {
+  createObservationsCh,
+  createTracesCh,
+} from "@/src/__tests__/async/repositories/clickhouse-helpers";
+import {
+  createObservation,
+  createTrace,
+} from "@/src/__tests__/fixtures/tracing-factory";
 
 describe("Create and get comments", () => {
   beforeEach(async () => await pruneDatabase());
   afterEach(async () => await pruneDatabase());
```

Comment:
This here is why the tests are failing. With this we remove models and prices from the db.

Reply:
Ah interesting - thank you! 
Do we need models and prices for most tests? If so, we could consider adding a flag to the `pruneDatabase` function that keeps models and prices by default, unless explicitly required them be deleted? Because behaviour in the comments test resulting in a failure of a test in `/public/observations` feels a little off :)

Reply:
I think we should not need pruneDatabase in the first place. We could build a helper to ensure prices and models are always there (this is the default in our database at all times in production).

Reply:
Fair point on not requiring pruneDatabase. I like the idea with a helper to ensure prices and models are always there 🙌

Reply:
There is also a helper to create orgs, projects and api keys one off https://github.com/langfuse/langfuse/blob/2f14d08569e60b2f36d25745d7d0a45811494f93/web/src/__tests__/test-utils.ts#L142